### ISSUE TITLE
Testing full logic with Mobx/NextJS

### ISSUE THREAD

**Issue by akve** [codeBelt/nextjs-mobx#42]
Hi Robert!

I've noticed your current obsession with NextJS+Mobx, and I share it as well. Found your repo and questioning if you've managed to proceed with TDD approach to testing logic completely, based on this tech stack. It seems viable, but seems like I still look for a shortcut - maybe someone has done this already... Probably you've moved further already in this direction, cause your boilerplate seem very-very close to test-first approach and separating logic completely from UI - but not yet there completely. 

So, probably you have any moves forward in applying more testing to this repository? 

Thanks for inspiration!
Alex

**Comment by codeBelt** [comment]
Yes @akve, I do love MobX and Next.js. I don't think I will work on this repo anymore because if I had time I would work on this one https://github.com/codeBelt/mobx-local-global-stores which is also Next.js and MobX. To be honest I've never done TDD so I couldn't give you insight on how to improve theses repo's.

**Comment by akve** [comment]
@codeBelt , thanks for the reply. I'm more obsessed now with the idea of TDD as a key to force separation of logic from framework and views. So I'll hopefully share with you what I'll come up with. Still based on NextJS and Mobx. Local stores are also important - but it's the idea that I've already implemented :) Cheers!

**Comment by codeBelt** [comment]
I would like to see what you come up!
